_includes/header.html

```diff
@@ -15,9 +15,20 @@
         </a>
         <div class="dropdown-menu">
           {% for sublink in navlink.submenu %}
+          {% if sublink.divideup %}
```

Comment:
For simplicity, I would remove these dividers. imho it does not help a lot, but the dividers need to be "maintained", which is a bit annoying.

Otherwise, if you want to divide items with subitems from items without subitems, you don't need the `divideup`/`dividedown` properties. These could be replaced with something like `if sublink.subsubmenu and "there is a prev item"` (sorry I dont have the jekyll syntax at the top of my head).

Reply:
The dividers should underline that the pages are thematically related. Another solution could be to indent the subpages, but not sure if this requires any maintance in the future. 

@The-Ludwig whats your opinion regarding this issue?

Reply:
I think automating it with jekyll syntax is a good idea. I will try to implement it. As it does not really need to be maintained, because it is just checking if the variable exists, I think the explicit version is ok for now, as the merge is more or less urgent...

Reply:
Ok: it was not that hard to do... forloop.first == false / forloop.last == false did the trick... see next commit...